src/utils/markdown-converter.ts

```diff
@@ -0,0 +1,18 @@
+import { NotionToMarkdown } from "notion-to-md";
+import type { TProcessedRichText } from "../types/processed-data.js";
+
+export async function convertToMarkdown(richText: TProcessedRichText): Promise<string> {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    const n2m = new NotionToMarkdown({} as any);
+
+    return await n2m.blockToMarkdown({
+        type: "paragraph",
+        paragraph: {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+            rich_text: richText as any,
```

Comment:
so here in the end you are casting the value to any regardless, so calling it processed rich text as done above is redundant